io.github.maddecoder.Classic-RBDOOM-3-BFG.yml

```diff
@@ -0,0 +1,136 @@
+app-id: io.github.maddecoder.Classic-RBDOOM-3-BFG
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "22.08"
+command: DOOMBFA.sh
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.mono6
+build-options:
+    append-path: /usr/lib/sdk/mono6/bin
+    append-ld-library-path: /usr/lib/sdk/mono6/lib
+    append-pkg-config-path: /usr/lib/sdk/mono6/lib/pkgconfig
+
+finish-args:
+- --device=all
+- --socket=wayland
+- --socket=x11
```

Comment:
also does it support wayland? If yes then line 16 is `fallback-x11`. otherwise remove wayland.

Reply:
Yes, yes it does

Reply:
done